### PR TITLE
Add translated demonyms in French.

### PR DESCRIPTION
**Pull request by alexsegura**
Hi,

First of all, thank you for this project 💯 

As suggested in #165, I added translated demonyms (in French). 
It is indexed by `f` or `m` to manage gender inflection. 
I decided to **add** a new key rather than changing the structure, to keep things backward compatible. Tell me if it's ok for you. 

I used a PHP script to decode/encode the `countries.json` file, so some indentation may have changed (I used `json_encode` with `JSON_PRETTY_PRINT` option). You can add `?w=1` to the GitHub URL to get rid of whitespace changes.

### CODE REVIEW

**Comment by alexsegura** [comment]
I see that `json_encode` has replaced `\u0020` by ` `.
I'm going to fix this.

File: countries.json [hunk 1]
```diff
@@ -23669,7 +24467,13 @@
         "landlocked": false,
         "borders": [],
         "area": 270467,
-        "flag": "\ud83c\uddf3\ud83c\uddff"
+        "flag": "\ud83c\uddf3\ud83c\uddff",
+        "demonyms": {
+            "fra": {
+                "f": "Neo-zelandaise",
+                "m": "Neo-zelandais"
```
Comment (mledoze) [inline countries.json]: Accents are missing here. Also `z` should be capitalized, see https://fr.wikipedia.org/wiki/N%C3%A9o-Z%C3%A9landais

File: countries.json [hunk 2]
```diff
@@ -24761,7 +25601,13 @@
             "IDN"
         ],
         "area": 462840,
-        "flag": "\ud83c\uddf5\ud83c\uddec"
+        "flag": "\ud83c\uddf5\ud83c\uddec",
+        "demonyms": {
+            "fra": {
+                "f": "Papouane-neoguineenne",
+                "m": "Papouane-neoguineen"
```
Comment (mledoze) [inline countries.json]: It should be `Papouasienne` and `Papouasien`, see https://fr.wikipedia.org/wiki/Papouasie-Nouvelle-Guin%C3%A9e

File: countries.json [hunk 3]
```diff
@@ -32942,7 +34082,13 @@
         "landlocked": false,
         "borders": [],
         "area": 389,
-        "flag": "\ud83c\uddfb\ud83c\udde8"
+        "flag": "\ud83c\uddfb\ud83c\udde8",
+        "demonyms": {
+            "fra": {
+                "f": "Saint-Vincentaise-et-Grenadine",
+                "m": "Saint-Vincentaise-et-Grenadin"
```
Comment (mledoze) [inline countries.json]: It should be `Vincentaise` and `Vincentais`, see https://fr.wikipedia.org/wiki/Saint-Vincent-et-les-Grenadines

**Comment by alexsegura** [comment]
@mledoze done!

**Comment by mledoze** [comment]
Thank you for the chnages. Also, could you please update the `README.md` file to add a description of the new `demonyms` property?

**Comment by alexsegura** [comment]
@mledoze done!

**Comment by mledoze** [comment]
Thank you for the update of README.md, it's perfect.

I will merge this as is for the moment but we will need to merge the `demonym` property into `demonyms` to avoid duplication.

**Comment by alexsegura** [comment]
> we will need to merge the demonym property into demonyms to avoid duplication

I didn't do this, because it would need a major version bump, but I noticed you just released a new major version `3.x` 🙂

**Comment by mledoze** [comment]
Yes I did because of the breaking changes made to the `currency` property. 

I don't mind to release a new major version, really, that's why we use semantic versioning.

**Comment by alexsegura** [comment]
Once it's merged, if you want I can take care of moving `demonym` to `demonyms.eng`, and removing `demonym`. 
Just open an issue and mention me.

**Comment by mledoze** [comment]
> Once it's merged, if you want I can take care of moving `demonym` to `demonyms.eng`, and removing `demonym`.
> Just open an issue and mention me.

I'd like that, thank you.

**Comment by mledoze** [comment]
Hello @alexsegura, I was going to release a new version 4.0 but it seems that some countries (51 exactly) are missing the `demonyms` property. Could you please send me a pull request to add them?

```bash
$ grep --count demonyms countries.json
199
```

Even if there are no demonyms for a country, the property must exist for each entry, even if it's empty.

Thank you

**Comment by mledoze** [comment]
Hello @alexsegura friendly ping 🙂 

Will you be able to add the missing demonyms? If no, I can take care of it.
Thank you for your help.

**Pull request merged**